src/cpu/o3/lsq_unit.hh

```diff
@@ -538,6 +538,9 @@ class LSQUnit
         /** Distribution of cycle latency between the first time a load
          * is issued and its completion */
         statistics::Distribution loadToUse;
+
+        /** Total number of squashed stores. */
```

Comment:
At minimum I think this comment is wrong.  First, the above changes are seeming to add stats when new loads or stores are added to the LSQ, but this comments says we're counting number of squashes.  Second, comment says squashed stores but counter is for loads and stores.